RELEASE/scripts/autoscend/auto_pre_adv.ash

```diff
@@ -501,6 +501,28 @@ boolean auto_pre_adventure()
 			}
 		}
 		if(itemDrop < itemNeed._float)
+		{
+			//if general item modifier isn't enough check specific item drop bonus
+			generic_t itemFoodNeed = zone_needItemFood(place);
+			generic_t itemBoozeNeed = zone_needItemBooze(place);
+			if(itemFoodNeed._boolean)
+			{
+				auto_log_debug("Trying food drop supplements");
+				//max at start of an expression with item and food drop is ineffective in combining them, have to let the maximizer try to add on top
+				addToMaximize("49food drop " + ceil(itemFoodNeed._float) + "max");
```

Comment:
why 49?

Reply:
because item gets 50, if the maximizer has to choose between food/item with same value. I don't think any source of item is only 1/50 worse than a food one but you could also change to 49.9 if you prefer